tests/sentry/auth/providers/github/test_client.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+from sentry.testutils import TestCase
+from sentry.utils.compat.mock import patch
+
+from sentry.auth.providers.github.client import GitHubClient
+
+
+class MockReq:
+    def __init__(self, status_code=200, content=None):
+        self.status_code = status_code
+        self.content = content
+
+
+class MockHttp:
+    def __init__(self):
+        self.calledArgs = []
+        self.calledKwargs = {}
+
+    def get(self, *args, **kwargs):
+        for arg in args:
+            self.calledArgs.append(arg)
+        self.calledKwargs = kwargs
+        return MockReq()
+
+
+@patch("sentry.utils.json.loads")
```

Comment:
There wasn't much prior art I could find for auth client testing, so please lmk if you have ideas for cleaning up this test.

Reply:
@slohmes I would think you could just use the mock responses library we use: https://github.com/getsentry/responses

Reply:
There's a library! Perfect, thank you!